src/map/navi.cpp

```diff
@@ -203,7 +181,8 @@ bool navi_path_search(struct navi_walkpath_data *wpd, const struct navi_pos *fro
 	tp[i].flag = SET_OPEN;
 	tpused[i] = 1 + (from->x << 16 | from->y);
 
-	heap_push_node(&g_open_set, &tp[i]); // Put start node to 'open' set
+	g_open_set.push_back(&tp[i]);
+	current = g_open_set.front();
```

Comment:
do we need to set current? it should be done in the infinite loop below

Reply:
just to escape warning from visual studio saying that I am referencing an uninitialized variable